waku/handshake.go

```diff
@@ -26,6 +64,7 @@ type statusOptions struct {
 	ConfirmationsEnabled *bool       `rlp:"key=3"`
 	RateLimits           *RateLimits `rlp:"key=4"`
 	TopicInterest        []TopicType `rlp:"key=5"`
+	keyTypeMapping       keyTypeMapping
```

Comment:
Isn't this presuming that all the keys are of the same type? What if some are `string` and some are `uint`?

Reply:
Or would there be a `statusOptionKeyType` for each `statusOptionKey`?

Reply:
> Isn't this presuming that all the keys are of the same type? What if some are string and some are uint?

This is not possible. The current version always sends keys as strings and this will always send keys as ints unless it received keys as strings then it will also reply with string keys.

Reply:
Ok. I will sort this.